ext/intl/dateformat/dateformat.stub.php

```diff
@@ -104,7 +104,7 @@ public function isLenient() {}
     public function format($datetime) {}
 
     /**
-     * @param IntlCalendar|DateTime $datetime
+     * @param IntlCalendar|DateTimeInterface $datetime
```

Comment:
Changes to the stub files require to generate the respective arginfo.h files. In this case run `build/gen_stub.php ext/intl/dateformat/dateformat.stub.php`, and commit the modified dateformat_arginfo.h.